torchrl/data/replay_buffers/rb_prototype.py

```diff
@@ -41,9 +41,9 @@ def __init__(
         pin_memory: bool = False,
         prefetch: Optional[int] = None,
     ) -> None:
-        self._storage = storage or ListStorage(max_size=1_000)
-        self._sampler = sampler or RandomSampler()
-        self._writer = writer or RoundRobinWriter()
+        self._storage = storage if storage is not None else ListStorage(max_size=1_000)
```

Comment:
Yeah for some reason it is not working. I guess it does `A or B`, which is a Boolean (not a storage or anything). 
in general I try to avoid using a non-Boolean as a Boolean, it brings surprising results way too often! Also it harms code readability: `if stuff else []` is less clear than `if stuff is not None else []`.